spec/kong_tests.conf

```diff
@@ -23,6 +23,7 @@ anonymous_reports = off
 
 lua_package_path = ?/init.lua;./kong/?.lua
 
+nginx_user = root
```

Comment:
This will make the test instances run as root, which is a no-no imo. We can avoid that change and not care about the "override" test, its goal is not to test that each and every property being overridden correctly.